src/foundation.cs

```diff
@@ -13571,39 +13652,38 @@ interface NSCondition : NSLocking
 
 	// Placeholer, just so we can start flagging things
 	interface INSFastEnumeration {}
-	
-#if MONOMAC
+		
 	partial interface NSBundle {
 		// - (NSImage *)imageForResource:(NSString *)name NS_AVAILABLE_MAC(10_7);
+		[NoiOS][NoMacCatalyst][NoWatch][NoTV]
```

Comment:
Shouldn't this be added to the type too?

Reply:
_no_ - And this cost me over a day. If you add [NoiOS] to NSBundle then all of the places that use any NSBundle API will error in strange and wonderful ways. 

The _main_ definition is here - https://github.com/xamarin/xamarin-macios/blob/main/src/foundation.cs#L9734-L9738 and works everywhere.

Reply:
Ah, I see, it's a partial definition 🤦 

We should merge these unnecessary partial definitions at some point.